### PR TITLE
docs(docs-infra): change link

### LINKED ISSUE #55687: ViewRef broken "Change detection usage" link doesn't work
### Describe the problem that you experienced

Clicking link "[Change detection usage](https://angular.dev/api/core/ChangeDetectorRefusage-notes)" in "[ViewRef](https://angular.dev/api/core/ViewRef)" bring up the "Page No Found 🙃" page.

### Enter the URL of the topic with the problem

https://angular.dev/api/core/ViewRef

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

_No response_

### Add a screenshot if that helps illustrate the problem

_No response_

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by vladboisa**
Fix url link for Button Change detection usage

Fixes #55687

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/core/src/linker/view_ref.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {ChangeDetectorRef} from '../change_detection/change_detector_ref';
 /**
  * Represents an Angular view.
  *
- * @see {@link ChangeDetectorRef#usage-notes Change detection usage}
+ * @see {@link ChangeDetectorRef?tab=usage-notes Change detection usage}
```
Comment (vladboisa) [inline packages/core/src/linker/view_ref.ts:14]: If doesn't work maybe this solution may help:
`@see [Change detection usage](/api/core/ChangeDetectorRef?tab=usage-notes)`

Comment (JeanMeche) [reply packages/core/src/linker/view_ref.ts:14]: This is a typescript issue, see microsoft/typescript#58259

See the workaround I used in #55553

Comment (vladboisa) [reply packages/core/src/linker/view_ref.ts]: > This is a typescript issue, see [microsoft/TypeScript#58259](https://github.com/microsoft/TypeScript/issues/58259)
> 
> See the workaround I used in #55553

Done, ty for advice!

**Review by bencodezen — APPROVED** [review]
Thanks for the PR @vladboisa!

**Review by JeanMeche — APPROVED** [review]

**Review by pkozlowski-opensource — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 825023801baf8dbd0bd87b7ec46a65e869e08adb.

**Pull request closed without merging**